csp-guru/datapackage.json

```diff
@@ -227,13 +197,13 @@
             "role": "author"
         },
         {
-            "title": "Lana Laura Ollier",
```

Comment:
You sure you want to kick both out? This is an iteration of the old data, no? So wouldn't they keep their share?

Reply:
Johan meinte ich soll uns für die neue version als autoren eintragen. die anderen werden ja auch acknowledged. Vermutlich ists auch iwie egal.

Reply:
Ich würde sie drin lassen. Aber das überlasse ich dir.